examples/tour/Cargo.toml

```diff
@@ -9,4 +9,13 @@ publish = false
 iced.workspace = true
 iced.features = ["image", "debug"]
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 tracing-subscriber = "0.3"
```

Comment:
`tracing-subscriber` not working on Wasm is quite annoying.

There should be a crate that puts everything together so we can avoid writing the conditional dependencies here. I may create that myself later.

Reply:
Yeah I was quite surprised when I found out that tracing-subscriber is actually causing the error.

While looking for a solution with `tracing-subscriber` I also found [tracing-web](https://github.com/WorldSEnder/tracing-web) which can be used in addition to get tracing_subscriber working on Wasm.

This still requires us to differentiate between Wasm or not, so I went the same route as the `integration` example did and simply put `console_log` there.